src/components/TheRouter.js

```diff
@@ -35,7 +35,18 @@ const routes = [
 	// reliably with router-link due to
```

Comment:
the router file is usually located into `/src/router/index.js` ;)

Reply:
Doesn't seem so usual to me 😉 

https://github.com/nextcloud/calendar/blob/master/src/router.js
https://github.com/nextcloud/notes/blob/master/src/router.js
https://github.com/nextcloud/bookmarks/blob/master/src/router.js

vs.

https://github.com/nextcloud/contacts/blob/master/src/router/index.js

But I agree, it is never else located in `src/components/TheRouter.js`. How should we do it?

Reply:
I would say it's fine as well in a router.js file, more clear ;)

I like having everything in separate folders, but feel free to choose :)
Both are fine imho (just not in components :wink:  )